monkestation/code/datums/quirks/negative_quirks.dm

```diff
@@ -63,11 +63,7 @@
 	addtimer(CALLBACK(src, PROC_REF(datacore_deletion)), 5 SECONDS)
 
 /datum/quirk/stowaway/proc/datacore_deletion()
-	var/mob/living/carbon/human/stowaway = quirk_holder
-	var/perpname = stowaway.name
-	var/datum/record/crew/record_deletion = find_record(perpname, GLOB.manifest.general)
-	SSjob.FreeRole(quirk_holder.mind.assigned_role)  //open their job slot back up
-	qdel(record_deletion)
+	quirk_holder.mind.remove_from_manifest()
```

Comment:
you should still prolly do the SSjob.FreeRole, as this _only_ removes their manifest records

Reply:
that was the intentional to remove that bit, talked about it with abraxis mentioned it in why its good for game. Was decided that stowaway probably shouldn't reopen slot for a number of small reasons he said it was good.

Reply:
- [ ] also not sure if I goofed but addtimer only allows for procs right? I tried to deproc it but didn't see anything like it. Just wanted to make sure I did it the best way

Reply:
@Shoddd you could do this:
```dm
addtimer(CALLBACK(quirk_holder.mind, TYPE_PROC_REF(/datum/mind, remove_from_manifest)), 5 SECONDS)
```

Reply:
Change made, thanks a ton. Don't tell pooba but your the best maint and so damn helpful. Have taught me a lot recently even if you didn't realize.